tests/test_resolver.py

```diff
@@ -814,26 +818,26 @@ def test_resolve_from_pex_ignore_errors(
     # type: (...) -> None
 
     # See test_resolve_from_pex_constraints above for the failure this would otherwise cause.
-    resolved_distributions = resolve_from_pex(
+    installed_distributions = resolve_from_pex(
         pex=pex_repository,
         requirements=["requests"],
         constraint_files=[create_constraints_file("urllib3==1.26.2")],
         interpreters=[py27],
         ignore_errors=True,
     )
-    resolved_distributions_by_key = {
-        resolved_distribution.distribution.key: resolved_distribution.distribution.as_requirement()
-        for resolved_distribution in resolved_distributions
+    installed_distributions_by_key = {
+        installed_distribution.distribution.key: installed_distribution.distribution.as_requirement()
+        for installed_distribution in installed_distributions
     }
-    assert len(resolved_distributions_by_key) > 1, "We should resolve at least requests and urllib3"
-    assert "requests" in resolved_distributions_by_key
-    assert Requirement.parse("urllib3==1.26.1") == resolved_distributions_by_key["urllib3"]
+    assert (
+        len(installed_distributions_by_key) > 1
+    ), "We should resolve at least requests and urllib3"
+    assert "requests" in installed_distributions_by_key
+    assert Requirement.parse("urllib3==1.26.1") == installed_distributions_by_key["urllib3"]
 
 
 def test_pip_proprietary_url_with_markers_issues_1415():
-    # https://<internal hostname>/pip/ray-0.8.4-cp37-cp37m-linux_x86_64.whl; sys_platform == 'linux'
-    # ray==0.8.4; sys_platform == 'darwin'
```

Comment:
Why remove this?

Reply:
It was stray from earlier today - I just used it as cut/paste from the original issue in order to recreate it in the unit test. I noticed it too late in the prior PR and so I saved a tree by waiting unitl this PR to kill it.